src/drawers/drawer.ts

```diff
@@ -111,16 +111,22 @@ export module Drawers {
      *
      * @returns {d3.Selection} the renderArea selection
      */
-    public _getRenderArea() {
+    public renderArea() {
```

Comment:
I can't comment on that LOC, but there is a `setup` method on the drawer, which just sets the `renderArea`. I suggest remove the setup call, and make this act as a setter as well